<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CClientReservationWidget</class>
 <widget class="QWidget" name="CClientReservationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="CAnimatedStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Забронированные билеты</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Поиск</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QGroupBox" name="groupBox_3">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="title">
                <string>ФИО клиента</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_3">
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item row="0" column="0">
                 <widget class="QLineEdit" name="leClient"/>
                </item>
                <item row="0" column="1">
                 <widget class="QToolButton" name="tbnClearClient">
                  <property name="icon">
                   <iconset resource="../clinet_res.qrc">
                    <normaloff>:/clientImages/clear.png</normaloff>:/clientImages/clear.png</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="1">
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>307</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Клиенты у которых есть забронированные билеты</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <widget class="QTreeWidget" name="twClients">
               <attribute name="headerDefaultSectionSize">
                <number>150</number>
               </attribute>
               <column>
                <property name="text">
                 <string>id</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>ФИО</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Дата рождения</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Логин/email</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Кол-во забронированных билетов</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pbnReservationManagement">
              <property name="text">
               <string>Управление бронью</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CAnimatedStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>canimatedstackedwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../clinet_res.qrc"/>
 </resources>
 <connections/>
</ui>
